### DISCUSSION TITLE
Screen off / Screensaver when nobody is there

### DISCUSSION THREAD

**Discussion by Rak3tenmann** [Screenly/Anthias#2079] [General]
I realized a small Infoscreen with a 5" Waveshare Touch.
Since this is just for me, I look for a way to switch off the screen if noone is watching. 
Or switch it off after x minutes and on again while touching.
Any ideas?

**Comment by vpetersson** [comment]
Anthias is running directly in framebuffer, so there's no such thing as a screensaver. It would possibly be doable once we've migrated to Wayland (but that's not the immediate roadmap).

**Reply by Rak3tenmann** [reply]
Got it - and thanks for the quick answer.
It just came to my mind, I could maybe force the Display to go to sleep.
see:
https://www.waveshare.com/wiki/5inch_DSI_LCD
But how to wakeup?
However: Great piece of software. 
But might be not fit for my usecase if the display burns 24h a day.
Thank you!

**Reply by vpetersson** [reply]
You might be able to do this with a cronjob of sorts, but it's not something that is supported in the native software.
